--- Template.php.original	2013-07-16 16:52:09.618593100 +0200
+++ Template.php	2013-07-16 17:01:52.326922100 +0200
@@ -1463,8 +1463,10 @@
 	 */
 	public function fetch_param($which, $default = FALSE)
 	{
-        if($which == 'status') {
-            echo "status";
+        if ($this->EE->extensions->active_hook('template_fetch_param') === TRUE)
+        {
+            $value = $this->EE->extensions->call('template_fetch_param', $this, $this->tagparts, $which, isset($this->tagparams[$which]) ? $this->tagparams[$which] : $default);
+            $this->tagparams[$which] = $value;
         }
 
 		if ( ! isset($this->tagparams[$which]))
